tests/ui/projects/test_projects_api.py

```
@@ -0,0 +1,96 @@
+        assert project
+
+        del json["metadata"]["name"]
+        with pytest.raises(MarshmallowValidationError):
+            # Marshmallow validation should fail
+            service.create(None, json)
+        with (
+            # disable the marshmallow validation
+            mock.patch.object(
+                ProjectServiceSchemaWrapper,
+                "load",
+                side_effect=lambda data, *a, **kw: (data, {}),
```

Ruff (0.14.14)

[warning] Unused lambda argument: `a`

(ARG005)